robottelo/hosts.py

```diff
@@ -253,6 +254,20 @@ def download_install_rpm(self, repo_url, package_name):
         if result.status != 0:
             raise ContentHostError(f'Failed to install {package_name} rpm.')
 
+    def download_repos(self, repo_name):
+        """Downloads the satellite or capsule repos on the machine
+        :param repo_name: satellite or capsule repo_name
+        """
+        release_version = f'{settings.server.version.release}.0'
```

Comment:
The `release` _could_ include the z-stream already, and this looks like its assuming only `X.Y` is set.  I'd be a bit more careful and check the length.

```
from packaging.version import Version
satver = Version(settings.server.version.release)
release_version = str(ver) if len(satver.release) >= 3 else f'{satver.major}.{satver.minor}.0'
```